kymatio/scattering1d/backend/torch_backend.py

```diff
@@ -127,5 +127,10 @@ def ifft(cls, x):
 
         return _ifft(x)
 
+    @classmethod
+    def swap_time_frequency(cls, x):
+        cls.complex_check(x)
+
+        return torch.transpose(x, dim0=-2, dim1=-3)
```

Comment:
maybe add a comment here stating that we expect `axis=-1` to be real/imag parts? hence we swap `-2` vs `-3`

Reply:
bon